tests/resources/modules/d8/woot/woot.install

```diff
@@ -1,8 +1,15 @@
 <?php
 
 /**
- * Failing update.
+ * Good update.
  */
 function woot_update_8101() {
-  throw new \Exception('8101 error');
+  return t('woot_update_8101');
+}
+
+/**
+ * Failing update.
+ */
+function woot_update_8102() {
+  throw new \Exception('8102 error');
```

Comment:
Any reason you added a new update?

Reply:
Yes, that was done on purpose to test the change I have made to src/Commands/core/UpdateDBCommands.php .

Without that change it will fail with exception from description.

Reply:
I created a new pull request https://github.com/drush-ops/drush/pull/3238 fix test changes only to prove the bug.